src/autopas/cells/FullParticleCell.h

```diff
@@ -250,15 +248,14 @@ class FullParticleCell : public ParticleCell<Particle> {
   /**
    * Storage of the molecules of the cell.
    */
-  StorageType _particles;
+  StorageType _particles{};
 
   /**
    * SoA buffer of this cell.
    */
-  SoA<SoAArraysType> _particleSoABuffer;
+  SoA<SoAArraysType> _particleSoABuffer{};
```

Comment:
_Side note:_

Why are these members public, but use the underscore syntax, indicating a private member.  
Maybe getters would be nicer. Or is it on purpose to avoid any overhead?

Reply:
puh... if I recall correctly this was initially a private member, hence the underscore. At some point, we made it public for ease of use. But I think we have a few naming inconsistencies like this one in the code and I think if we want to clean this up it should be a dedicated branch.